services/src/main/java/org/jboss/windup/web/services/model/AnalysisContext.java

```diff
@@ -273,6 +276,21 @@ public void removeApplication(RegisteredApplication application)
         this.getApplications().remove(application);
     }
 
+    public AnalysisContext clone()
+    {
+        AnalysisContext clone = new AnalysisContext(this.migrationProject);
```

Comment:
I am curious what the advantage to this is, vs just evicting it from the session and setting the id to null? It seems like that would do the same thing, since this isn't a deep clone.